README.md

```diff
@@ -11,9 +11,9 @@ What you need to run this app:
 - `node` and `npm` (`brew install node`)
 
   - Ensure you're running the latest versions Node `v16.x.x`+ and NPM `8.x.x`+
-- A [MongoDB](https://www.mongodb.com/docs/manual/administration/install-community/) instance running on your local machine
+- A version 6.0+ [MongoDB](https://www.mongodb.com/docs/manual/administration/install-community/) instance running on your local machine
```

Comment:
Note that MongoDB 6.0+ required, so that scripts can use `mongosh` instead of `mongo`

Reply:
Noted!